l10n_fr_fec/wizard/fec.py

```diff
@@ -182,8 +171,7 @@ def generate_fec(self, cr, uid, ids, context=None):
             'type': 'ir.actions.act_window',
             'view_mode': 'form',
             'res_model': self._name,
-            'res_id': ids[0],
+            'res_id': self[0].id,
```

Comment:
I see record set as it can be 2 things in one. Making this clear with `ensure_one` here you could consider your self is a single record.
But ok this is a matter of taste.